zmqw/zmqw.c

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
Please run `clang-format` over this whole file:
http://docs.frrouting.org/projects/dev-guide/en/latest/workflow.html#code-formatting